src/main/java/com/mraof/minestuck/util/MSTags.java

```diff
@@ -158,4 +166,39 @@ private static TagKey<Biome> tag(String name)
 			return TagKey.create(Registry.CONFIGURED_STRUCTURE_FEATURE_REGISTRY, new ResourceLocation(Minestuck.MOD_ID, name));
 		}
 	}
+	
+	public static boolean blockExistsInTag(TagKey<Block> blockTag, Block block)
+	{
+		for(Block blockIterate : getBlocksFromTag(blockTag))
+			if(blockIterate == block)
+				return true;
+		
+		return false;
+	}
```

Comment:
Replace usage of this with `BlockState.is()`.

Reply:
I may be confused, but I do not see that method in the BlockState class

Reply:
Considering how short that version is, and the fact that blockExistsInTag was unused, I just went and deleted it